### PR TITLE
Cortex Data Lake - Indicators Hunting Playbooks

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5010?filter=-1)

## Description
New playbooks for indicators hunting within Cortex Data Lake. 

## Screenshots
'Cortex Data Lake - Traffic Indicators Hunting' playbook:
Cortex_Data_Lake_-_Traffic_Indicators_Hunting

'Cortex Data Lake - File Indicators Hunting' playbook:
Cortex_Data_Lake_-_File_Indicators_Hunting

'Cortex Data Lake - Indicators Hunting' playbook:
Cortex_Data_Lake_-_Indicators_Hunting

### CODE REVIEW

File: Packs/CortexDataLake/ReleaseNotes/1_4_0.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+#### Playbooks
+##### New: Cortex Data Lake - Traffic Indicators Hunting
```
[the remark this answers is not in the document]
Comment (TalNos) [reply Packs/CortexDataLake/ReleaseNotes/1_4_0.md]: done

File: Packs/CortexDataLake/Playbooks/Cortex_Data_Lake_-_File_Indicators_Hunting.yml [hunk 2]
```diff
@@ -0,0 +1,1812 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "333":
+    id: "333"
+    taskid: ff5e0aab-37dc-42b0-8d2e-2370443306ab
+    type: playbook
+    task:
+      id: ff5e0aab-37dc-42b0-8d2e-2370443306ab
+      version: -1
+      name: Convert file hash to corresponding hashes
```
Comment (altmannyarden) [inline Packs/CortexDataLake/Playbooks/Cortex_Data_Lake_-_File_Indicators_Hunting.yml:1457]: what is the playbook (missing from the PR) and also Im not sure its neded

**Comment by ShirleyDenkberg** [comment]
@altmannyarden Doc review completed.

File: Packs/CortexDataLake/Playbooks/Cortex_Data_Lake_-_File_Indicators_Hunting.yml (hunk 2 above)
Comment (TalNos) [reply Packs/CortexDataLake/Playbooks/Cortex_Data_Lake_-_File_Indicators_Hunting.yml:1457]: The 'Convert file hash to corresponding hashes' playbook is part of the 'Common Playbooks' pack and is used to convert MD5 and SHA1 hashes to SHA256 hashes.

As 'cdl-query-threat-logs' and 'cdl-query-file-data' accept only SHA256 hashes, I decided to add 'Convert file hash to corresponding hashes' as a sub-playbook, so MD5 and SHA1 inputs will also be supported.

Comment (TalNos) [reply Packs/CortexDataLake/Playbooks/Cortex_Data_Lake_-_File_Indicators_Hunting.yml:1457]: @altmannyarden 
Let me know if you want me to remove this sub-playbook. If so, I will have to remove the MD5 and SHA1 inputs as well.

File: Packs/CortexDataLake/Playbooks/Cortex_Data_Lake_-_Traffic_Indicators_Hunting_README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+| start_time | Specify the query start time at which to perform a search within Cortex Data Lake.<br/><br/>For example, start_time="2018-04-26 00:00:00" |  | Optional |
+| end_time | Specify the query end time at which to perform a search within Cortex Data Lake.<br/><br/>For example, end_time="2018-04-26 00:00:00" |  | Optional |
+| limit | The maximum number of logs to return. <br/>Default is 10. |  | Optional |
+| fields | Select the fields you wish to be included in the query results. <br/>Selection can be "all" \(same as \*\) or a comma-separated list of specific fields in the table.<br/><br/>Separate multiple search values by commas only \(without spaces or any special characters\). |  | Optional |
+| action | Filter network traffic logs that should be retrieved from Cortex Data Lake based on firewall action.<br/><br/>Separate multiple search values by commas only \(without spaces or any special characters\). |  | Optional |
```
[the remark this answers is not in the document]
Comment (TalNos) [reply Packs/CortexDataLake/Playbooks/Cortex_Data_Lake_-_Traffic_Indicators_Hunting_README.md]: @altmannyarden - see my request for an argument that enables users to filter logs according to traffic direction in the [CIAC-5429](https://jira-hq.paloaltonetworks.local/browse/CIAC-5429) Jira ticket.

File: Packs/CortexDataLake/Playbooks/Cortex_Data_Lake_-_File_Indicators_Hunting.yml (hunk 2 above)
Comment (TalNos) [reply Packs/CortexDataLake/Playbooks/Cortex_Data_Lake_-_File_Indicators_Hunting.yml]: done

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**